akka-cluster/src/main/scala/akka/cluster/routing/ClusterRouterConfig.scala

```diff
@@ -46,33 +55,46 @@ final case class ClusterRouterGroupSettings(
   totalInstances:    Int,
   routeesPaths:      immutable.Seq[String],
   allowLocalRoutees: Boolean,
-  useRole:           Option[String]) extends ClusterRouterSettingsBase {
+  useRoles:          Set[String]) extends ClusterRouterSettingsBase {
```

Comment:
I am aware of binary incompatibility and @patriknw also mentioned it, but I do not know how to resolve that issue. Would you mind giving me some pointers ?